docs/filters/truncate.md

```diff
@@ -42,7 +42,7 @@ Local install:
 export default {
   computed: {
     text() {
-      return 'Orion\'s sword hundreds of thousands cosmic ocean as a patch of light the ash of stellar alchemy quasar. Network of wormholes paroxysm of global death vanquish the impossible kindling the energy hidden in matter hydrogen atoms invent the universe.';
+      return 'Orion\\\'s sword hundreds of thousands cosmic ocean as a patch of light the ash of stellar alchemy quasar. Network of wormholes paroxysm of global death vanquish the impossible kindling the energy hidden in matter hydrogen atoms invent the universe.';
```

Comment:
I don't understand. Shouldn't the one backslash work to escape the single quote? I actually don't know what 3 backslashes would do.

Reply:
I would also support setting the ESLint rule `"avoidEscape": true` or `"allowTemplateLiterals": true`

https://eslint.org/docs/rules/quotes#avoidescape
https://eslint.org/docs/rules/quotes#allowtemplateliterals

Reply:
* `'aaa's'` => Breaks in Editor => shows as `'aaa's'` in Vue live  => Vue Live gives `Unexpected token (5:20)`
* `'aaa\'s'` => Works in Editor => shows as `'aaa's'` in Vue live => `Unexpected token (5:20)`
* `'aaa\\\'s'` => Works in Editor => shows as `'aaa\'s'` in Vue live => Displays without errors

Reply:
Oh interesting. Do you think this is a better solution than modufying `avoidEscape` or `allowTemplateLiterals`?